### PR TITLE
Update changelog.rst

### PR DESCRIPTION
**Pull request by jacoffey3**

### CODE REVIEW

**Comment by jacoffey3** [comment]
[KubOS 1.5](https://trello.com/c/NnPrhk47/44-kubos-15)

File: docs/dev-docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,21 @@
 Kubos Changelog
 ===============
 
+v1.5.0 - Sep 7th 2018
+---------------------
+
+- `Added a community Trello board for contributors and KubOS team members <https://trello.com/b/pIWxmFua/kubos-community>`__
+- `The file transfer service and client have both been rewritten into Rust <final link to file service here>`__
+- Added documentation for how to handle the deployment hold time countdown 
+- Updated the app APIs to commonize behaviour between the `Python <https://github.com/kubos/kubos/tree/master/apis/app-api/python>`__ and `Rust <https://github.com/kubos/kubos/tree/master/apis/app-api/rust>`__ versions
```
Comment (CatGarab) [inline docs/dev-docs/changelog.rst]: nit: behaviour -> behavior ('Murica :us: :eagle: )

File: docs/dev-docs/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,21 @@
 Kubos Changelog
 ===============
 
+v1.5.0 - Sep 7th 2018
+---------------------
+
+- `Added a community Trello board for contributors and KubOS team members <https://trello.com/b/pIWxmFua/kubos-community>`__
+- `The file transfer service and client have both been rewritten into Rust <final link to file service here>`__
```
Comment (CatGarab) [inline docs/dev-docs/changelog.rst]: The URL for the service will be https://github.com/kubos/kubos/tree/master/services/file-service

File: docs/dev-docs/changelog.rst [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+- Added documentation for how to handle the deployment hold time countdown
```
Comment (CatGarab) [inline docs/dev-docs/changelog.rst]: nit: Add link to doc

File: docs/dev-docs/changelog.rst [hunk 4]
```diff
@@ -1,6 +1,21 @@
...
+- `The file transfer service and client have both been rewritten into Rust <final link to file service here>`__
+- Added documentation for how to handle the deployment hold time countdown 
+- Updated the app APIs to commonize behaviour between the `Python <https://github.com/kubos/kubos/tree/master/apis/app-api/python>`__ and `Rust <https://github.com/kubos/kubos/tree/master/apis/app-api/rust>`__ versions
+- Added example mission applications for both `Rust <https://github.com/kubos/kubos/tree/master/examples/rust-mission-app>`__ and `Python <https://github.com/kubos/kubos/tree/master/examples/python-mission-app>`__
+- `Added a verbose example mission application in Python for mission design <https://github.com/kubos/kubos/tree/master/examples/python-mission-application>`__
+
+Bug Fixes:
+~~~~~~~~~~
+
+- Updated usage of the Rust failure crate
```
Comment (plauche) [inline docs/dev-docs/changelog.rst]: Specifically we have standardized our usage of failure to use the 0.1.2 version.

File: docs/dev-docs/changelog.rst (hunk 1 above)
Comment (jacoffey3) [reply docs/dev-docs/changelog.rst]: 🤣

**Pull request merged**